lib/ansiblelint/rules/PackageIsNotLatestRule.py

```diff
@@ -29,5 +29,10 @@ class PackageIsNotLatestRule(AnsibleLintRule):
     tags = ['repeatability']
 
     def matchtask(self, file, task):
-        return (task['action']['__ansible_module__'] in ['yum', 'apt', 'package'] and
-                task['action'].get('state') == 'latest')
+        return (task['action']['__ansible_module__'] in ['yum', 'apt', 'dnf', 'homebrew', 'pacman',
+                                                         'homebrew_cask', 'openbsd_package',
```

Comment:
I was checking each of them for a `state` parameter and forgot that I need to verify that each `state` parameter needs to be able to take `latest`.